manifests/ruby.yml

```diff
@@ -345,6 +345,7 @@ tests/:
       Test_Environment: missing_feature
       Test_TelemetryInstallSignature: missing_feature
       Test_TelemetrySCAEnvVar: v2.1.0
+      Test_Consistent_Configs: missing_feature
```

Comment:
CI is complaining about this not being sorted alphabetically, so you'll need to move this to be the first entry of the `test_telemetry.py` grouping here